src/Map/Map.jsx

```diff
@@ -0,0 +1,63 @@
+import React, {Component, PropTypes} from 'react'
+import ReactHighcharts from 'react-highcharts'
+import {LayerGroup, LayersControl, Map as LeafMap, Marker, Polyline, Popup, TileLayer, ZoomControl} from 'react-leaflet'
```

Comment:
* Put each of these on a new line:
```javascript
import {
  LayerGroup as LeafletLayerGroup,
  LayersControl,
  Map as LeafletMap,
  Marker,
  Polyline,
  Popup,
  TileLayer,
  ZoomControl
} from 'react-leaflet'
```

* note that I renamed `LeafMap` to `LeafletMap` and also added an alias for `LayerGroup` to be called `LeafletLayerGroup`